src/styles/partials/card.scss

```diff
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid $concrete-color;
+  background: $white;
+  transition: transform, 0.3s;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px;
+  }
+
+  &__title {
+    display: flex;
+    width: 166px;
+    flex-direction: column;
+    margin: 0;
+
+    &-label {
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 18px;
+      margin: 8px 0 0;
+      color: $main-accent-color;
+    }
+
+    &-code {
+      color: $secondary-color;
+      font-family: $body-font;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      width: 166px;
+      margin-top: 4px;
+      margin-bottom: 16px;
+    }
+  }
+
+  &__reviews {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: flex-end;
+
+  }
+
+  &__reviews-counter {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-amount {
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $blue-accent-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-transform: uppercase;
+    color: $white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    margin: 16px;
+  }
+
+  &__button:hover {
+    background-color: $white;
+    color: $blue-accent-color;
+    border: 1px solid $blue-accent-color;
+    width: 164px;
+    height: 40px;
+  }
```

Comment:
Nesting, also there is no need to change `width` or `height` 
`border` should be applied on `&__button`: `border: 1px solid transparent`, and then you only change `border-color` on `:hover`